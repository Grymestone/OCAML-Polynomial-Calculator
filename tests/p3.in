9a^4 + 32a^2 - 31a^2 
