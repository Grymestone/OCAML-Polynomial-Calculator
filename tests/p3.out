(((9*(a)^4)+(32*(a)^2))-(31*(a)^2))
1x^2 + 9x^4